services/core-web/src/components/mine/NoticeOfWork/MineNOWApplications.js

```diff
@@ -102,13 +104,28 @@ export class MineNOWApplications extends Component {
   };
 
   render() {
+    const isMajorMine = this.props.mines[this.props.mineGuid].major_mine_ind;
+    const title = isMajorMine ? "Permit Applications" : "Notice of Work Applications";
     return (
       <div className="tab__content">
         <div>
-          <h2>Notice of Work Applications</h2>
-          <Divider />
+          <h2>{title}</h2>
+          {isMajorMine && (
```

Comment:
update to use AuthWrapper and Should this also have the "permit_create" permissions?

Reply:
yup, done.